saml_sp.js

```diff
@@ -453,6 +453,11 @@ function saveSAMLAttributes(r, root) {
         if (attrs.hasOwnProperty(attributeName)) {
             var attributeValue = attrs[attributeName];
 
+            /* If the attribute name is a URI, take only the last part after the last "/" */
+            if (attributeName.includes("http://") || attributeName.includes("https://")) {
+                attributeName = attributeName.split("/").pop();

```

Comment:
may be better a bit:

attributeName = attributeName.substring(attributeName.lastIndexOf('/')+1);